release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -781,6 +853,11 @@ In the following tables, features are marked with the following statuses:
 |====
 |Feature |4.17 |4.18 |4.19
```

Comment:
I think we want to add "Increasing the maximum number of nodes per volume for vSphere" as TP in the table.

Reply:
Done.